src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/queries/HearingForListingQueries.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.sandl.snlevents.repository.queries;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class HearingForListingQueries {
+
+    private final String hearingForListingSelectCount = "SELECT Count(*) ";
+    private final String hearingForListingSelectHearings = "SELECT h.id, case_number, case_title, "
+        + "ct.code as case_type_code, "
+        + "ct.description as case_type_description, ht.code as hearing_type_code, "
+        + "ht.description as hearing_type_description, (duration * 1000000000) as duration, "
+        + "schedule_start, schedule_end, version, "
+        + "priority, communication_facilitator, reserved_judge_id, p.name as reserved_judge_name, number_of_sessions, "
+        + "h.status, is_multisession ";
+
+    private final String hearingForListingQueryBody = "FROM hearing h "
```

Comment:
why not merge them into one constant?

Reply:
It's used in 'totalcount' query and 'entities' query